### ISSUE TITLE
dbt-duckdb failing to run

### ISSUE THREAD

**Issue by matsonj** [matsonj/nba-monte-carlo#76]
probably something with duckdb 0.7.0

**Comment by matsonj** [comment]
```
Runtime Error
  Binder Error: Catalog "main" does not exist!
```

cc @jwills I think i can fix this by upgrading the versions, but as an FYI it looks like a regression

**Comment by jwills** [comment]
@matsonj I am on it

**Comment by matsonj** [comment]
Ok the fix is pinning ```duckdb==0.6.1``` in the meltano.yml file, so that means I think the entire adapter could have an issue?

**Comment by jwills** [comment]
yah; the TL;DR here is that the new attach functionality in 0.7.0 means that the `database` part of the `database.schema.relation` naming scheme is now load-bearing as opposed to, well, not load-bearing. You could also work around it by updating the dbt profile to set the `database` field to e.g. `memory` if you are using the `:memory:` path; I have a PR to fix this by auto-setting the `database` property for you based on the path: https://github.com/jwills/dbt-duckdb/pull/98

**Comment by matsonj** [comment]
Yeah that makes sense, I have some general refactoring I need to do in this project so I will pick up the fix and support 0.7.0 in the next few days.
